photoeditor/src/main/java/ja/burhanrashid52/photoeditor/ZedgeImageFilterView.java

```diff
@@ -0,0 +1,209 @@
+package ja.burhanrashid52.photoeditor;
```

Comment:
I think it would be a good idea to move this class into the package net.zedge.photoeditor,